app/src/main/java/com/tohami/newsapi/ui/news/NewsViewModel.java

```diff
@@ -0,0 +1,95 @@
+package com.tohami.newsapi.ui.news;
+
+import androidx.lifecycle.LiveData;
+import androidx.lifecycle.MediatorLiveData;
+import androidx.lifecycle.MutableLiveData;
+
+import com.tohami.newsapi.BuildConfig;
+import com.tohami.newsapi.Constants;
+import com.tohami.newsapi.base.BaseViewModel;
+import com.tohami.newsapi.data.model.DataModel;
+import com.tohami.newsapi.data.model.NewsArticle;
+import com.tohami.newsapi.data.model.NewsResponse;
+import com.tohami.newsapi.data.repos.NewsRepository;
+import com.tohami.newsapi.factory.RepositoryFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class NewsViewModel extends BaseViewModel {
```

Comment:
Please check this article 
https://developer.android.com/jetpack/docs/guide

Reply:
done